### ISSUE TITLE
Implement babel.

### ISSUE THREAD

**Issue by codeviking** [allenai/syrup#13]
Babel is a compiler which takes ES6 syntax and compiles it to ES5 compatible code.  The S2 project has used it extensively and recommends we adopt it for ES6 syntax support (it also supports JSX transformation out of the box).

https://babeljs.io/

**Comment by codeviking** [comment]
@markschaake Curious to get your input here.  There's two ways to set this up.

1) It can be added as an explicit transform when we bundle the JS.  For instance:

```javascript
var bundler = browserify();
bundler.transform(babelify());
```

2. It can be configured in the package.json of the associated project:

```javascript
{
   ...
    "browserify" : "transforms": ["babelify"],
    ...
}
```

Option #1 means it doesn't have to be configured / enabled.  I could add am option for disabling it (`disableBabel`).

Option #2 means babel is a per-project configuration parameter.  It means their is more setup, but if a user doesn't need it or wants to use Angular w/o Babel they could.

What's your vote?

**Comment by codeviking** [comment]
Resolved as of 0.1.5.

**Comment by codeviking** [comment]
It's now included by default, as is stringify.
